AngularApp/src/app/modules/dashboard/dashboard.module.ts

```diff
@@ -82,6 +83,11 @@ export const DashboardModuleRoutes: ModuleWithProviders = RouterModule.forChild(
             component: TabDevelopComponent
           }
           ,
+          {
+            path: 'changelist',
+            component: TabDetectorChangelistComponent
+          }
```

Comment:
can you fix the ',' being on its own line, should just be after '}'. You can just do a `shift + alt + f` to format the page and it should fix it.